judo-ui-react/src/main/resources/actor/src/theme/fonts.tsx.hbs

```diff
@@ -0,0 +1,7 @@
+// Fonts have to be imported to notify bundler to include
+// import PlusJakarta from '../custom/assets/fonts/PlusJakartaSans-VariableFont_wght.ttf';
```

Comment:
If we don't ship additional fonts, then we don't need these comments, right? :)

Reply:
It's an example